doc/SCALING.md

```diff
@@ -0,0 +1,106 @@
+
+Scaling-up the number of instances in the `stunner` deployment should be done based on the CPU usage of the replicas. As it was mentioned in the [section above](#scaling-with-stunner) the `HorizontalPodAutoscaler` is using the requested resources to determine when to scale up.
+
+### Scaling-down (in)
+
+When removing existing instances from the `stunner` replica set there are some things to keep in mind. What happens to the existing calls on a to-be-removed pod? Is there a way to keep them? 
+
+In worst case scenario all call connections will be dropped and the pod terminates itself, thus we lost all running connections/calls on the removed (scaled down) STUNner instance. A slightly better scenario is that the [ICE restart](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/restartIce) mechanism will kick at client side, basically it will reset the current ICE candidates and reconnect again the same way it did initially but it takes a second or two and not even supported by all browsers. 
+
+The third option is that STUNner [has full support](#more-on-the-graceful-shutdown) for graceful shutdown in Kubernetes. This means that `stunner` pods will remain alive as long as there are active allocations via the embedded TURN server, and a pod will automatically remove itself once all allocations through it are deleted or timed out. It is important that the *terminating* pod will not be counted in by the `HorizontalPodAutoscaler` as a running replica, thus its CPU usage won't be taken into account either. 
+
+#### More on the graceful shutdown
+
+Note that the default TURN refresh lifetime is 10 minutes so STUNner may remain alive well after the last client goes away. This occurs when an UDP allocation is left open by a client (spontaneous UDP client-side connection closure cannot be reliably detected by the server). In such cases, after 10 mins the allocation will timeout and get deleted, which will then let `stunnerd` to terminate. 
+This feature enables the full support for graceful scale-down: the user can scale the number of `stunner` instances up and down as they wish and no harm should be made to active client connections meanwhile. 
+Caveats: 
+- currently the max lifetime for `stunner` to remain alive after deleted in 1 hour: this means that `stunner` will remain active only for 1 hour after it has been deleted/scaled-down. You can always set this in by adjusting the `terminationGracePeriod` on your `stunnerd` pods.
+- if there are active (or very recent) TURN allocations then the `stunner` pod may refuse to be removed after a kubectl delete. Use `kubectl delete pod --grace-period=0 --force -n stunner stunner-XXX` to force removal.
+
+
+## Example
+
+In this part we will walk you through a simple example on how to scale your `stunner` deployment. 
+
+A simple `HPA` config for scaling `stunner` would be:
```

Comment:
We should refer the reader to a doc on how to setup HPA. Is HPA available on a default K8s install?